pandas/io/formats/style.py

```diff
@@ -540,6 +627,7 @@ def render(self, **kwargs) -> str:
         self._compute()
         # TODO: namespace all the pandas keys
         d = self._translate()
+        d = self.tooltips._translate_tooltips(self.data, self.uuid, d)
```

Comment:
this is best before the update(), because it is a direct extension of the previous _translate() method.

Reply:
this is *very* confusing  here, why is this not part of _translate itself?